### PR TITLE
Add grad student career management plan

### LINKED ISSUE #20: Write career management plan for part-time grad students.
And possibly for the assessor and instructional designer? (Hard to know whether to do that if they're going to be professionals rather than grad students?)  There's a fixme in proposal.tex.

### PR DESCRIPTION
**Pull request by ethanwhite**
Fixes #20.

### CODE REVIEW

File: proposal.tex [hunk 1]
```diff
@@ -356,7 +356,24 @@ \section{Broader Impact}
...
+undergraduates at the different universities will each be paired with
+a local faculty mentor. The faculty mentor will meet regularly with
+the graduate student to discuss and problem solve any issues that the
+graduate student or undergraduates are having and to provide active
+mentoring on how to train students in computational approaches.
+
+In addition to engaging with the graduate students on their mentoring
+of the undergraduates, the faculty mentors will also serve as a
+mentors for computational aspects of the graduate students' research
+and careers. In many areas of science computationally minded students
```
Comment (rachelslaybaugh) [inline proposal.tex]: ...science, computationally-minded students...

File: proposal.tex [hunk 2]
```diff
@@ -356,7 +356,24 @@ \section{Broader Impact}
...
+are located in labs where the PIs do not have strong computational
+backgrounds. This means that they do not have a mentor teach them
```
Comment (rachelslaybaugh) [inline proposal.tex]: ...mentor to teach...

File: proposal.tex [hunk 3]
```diff
@@ -356,7 +356,24 @@ \section{Broader Impact}
...
+about good computational practice in research. In addition, they do
+not have someone to discuss computational careers with, thus limiting
+their exposure to career paths outside of academia. Because the
+faculty mentors will have strong computational backgrounds themselves
```
Comment (rachelslaybaugh) [inline proposal.tex]: ...themselves, faculty...

File: proposal.tex [hunk 4]
```diff
@@ -356,7 +356,24 @@ \section{Broader Impact}
...
+they can fill this void for computationally minded students.
```
Comment (rachelslaybaugh) [inline proposal.tex]: computationally-minded

File: proposal.tex [hunk 5]
```diff
@@ -356,7 +356,24 @@ \section{Broader Impact}
 
 \section{Career Management Plan}
 
-\fixme{EW}{Talk about the part-time grad students we're planning to hire.}
+The graduate students who are serving as mentors for the
+undergraduates at the different universities will each be paired with
+a local faculty mentor. The faculty mentor will meet regularly with
+the graduate student to discuss and problem solve any issues that the
+graduate student or undergraduates are having and to provide active
```
Comment (rachelslaybaugh) [inline proposal.tex]: having, and to

**Comment by ethanwhite** [comment]
Fixed.

**Pull request merged**